examples/README.rst

```diff
@@ -0,0 +1,112 @@
+Bespoke linear operators
+=================================
+In this section we provide a range of pedagogical example applications of 
+``optimusprimal`` both on 1D signals (*e.g.* time series datasets) and 2D 
+signals (*e.g.* image processing datasets). In each case there are many 
+commonalities. The canonical structure of these applications is 
+
+**import/simulate data** :math:`\rightarrow` 
+**define a data-fidelity (likelihood)** :math:`\rightarrow` 
+**define a regulariser (prior)** :math:`\rightarrow` 
+**run the primal-dual solver**.
+
+Custom Measurement operators :math:`\Phi`
+-----------------------------------------
+For each application we adopt a standard de-noising setting, in which the 
+measurement (sensing) operator is simply the identity matrix. In practice 
+this operator is customized for a particular application. To do his one 
+simply needs to define a linear operator class which has the following 
+structure
+
+.. code-block:: python 
+
+    class my_custom_linear_operator:
+        """A custom linear operator e.g. a custom measurement operator"""
+
+        def __init__(self, ... ):
+            """Initialise the operator with any necessary parameters"""
+            self.(...) = (...)
+        
+        def dir_op(self, x):
+            """Forward linear operator 
+
+            Args: 
+
+                x (np.ndarray): vector this operator should be applied to
+            
+            Returns:
+
+                f(x) (np.ndarray): Forward operator applied to x
+            """
+            return f(x)
+        
+        def adj_op(self, x):
+            """Adjoint linear operator 
+
+            Args: 
+
+                x (np.ndarray): vector this adjoint operator should be applied to
+            
+            Returns:
+
+                f^T(x) (np.ndarray): Forward operator applied to x
+            """
+            return f^T(x)
+
+this function is then passed to your data-fidelity term instead of 
+``linear_operator.identity()`` *e.g.*
+
+.. code-block:: python 
+
+    # What was 
+    prox_operators.l2_ball( ... , Phi=linear_operator.identity())
+    
+    # Becomes
+    prox_operators.l2_ball( ... , Phi=my_custom_linear_operator)
+
+Custom Wavelet Transform :math:`\Psi`
+-------------------------------------
+Though PyWavelet has a large `catalogue <https://tinyurl.com/5n7wzpmb>`_ of Euclidean 
```

Comment:
This can probably be a nicely formatted link

Reply:
As we're swapping to nblinks this will be removed.